plugin/slime.vim

```diff
@@ -77,7 +77,9 @@ function! s:TmuxConfig() abort
   if !exists("b:slime_config")
     let b:slime_config = {"socket_name": "default", "target_pane": ":"}
   end
-  let b:slime_config["socket_name"] = input("tmux socket name or absolute path: ", b:slime_config["socket_name"])
+  if !(exists("g:slime_dont_ask_tmux_socket_name") && g:slime_dont_ask_tmux_socket_name)
```

Comment:
This was not supposed to be in this PR - I just pushed some unrelated change and ended up here :P Will take this out